### PR TITLE
Add in Traffic Manager deployment pattern docs

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2/docs-apim/issues/4714

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Configuring a Distributed API-M Deployment
+# Configuring a Distributed API-M Deployment with Gateway and Control Plane
 
 The WSO2 API-M server can be deployed as an [all-in-one deployment]({{base_path}}/install-and-setup/setup/single-node/all-in-one-deployment-overview) or as a distributed deployment. In the distributed setup, the [API-M server profiles]({{base_path}}/install-and-setup/setup/distributed-deployment/product-profiles) are deployed as separate API-M nodes. 
 
-Given below are the API-M nodes you can have in a distributed deployment.
+Given below are the API-M nodes you can have in a distributed deployment with Traffic Manager included within the Control Plane.
```
Comment (tharikaGitHub) [inline en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md:5]: Shall we change this to "Given below are the API-M nodes you can have in a distributed deployment by default." @tharindu1st wdyt?

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 2]
```diff
@@ -80,308 +74,302 @@ See the instructions on [configuring the API Gateway]({{base_path}}/api-analytic
...
+wss_endpoint = "wss://gw.wso2.com:8099"
+http_endpoint = "http://gw.wso2.com:8280"
+https_endpoint = "https://gw.wso2.com:8243"
+
+[apim.cors]
+allow_origins = "*"
+allow_methods = ["GET","PUT","POST","DELETE","PATCH","OPTIONS"]
+allow_headers = ["authorization","Access-Control-Allow-Origin","Content-Type","SOAPAction"]
+allow_credentials = false
+```
```
Comment (tharikaGitHub) [inline en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md:372]: Let's remove the Traffic Manager startup command section from this document.

File: en/mkdocs.yml [hunk 3]
```diff
@@ -2112,5 +2113,5 @@ extra:
         - type: linkedin
           link: https://www.linkedin.com/company/wso2
     site_version: 4.0.0
-    #base_path: http://localhost:8000
-    base_path: https://apim.docs.wso2.com/en/4.0.0
+    base_path: http://localhost:8000
+    #base_path: https://apim.docs.wso2.com/en/4.0.0
```
Comment (tharikaGitHub) [inline en/mkdocs.yml:2117]: Let's uncomment the base path and comment localhost.

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md (hunk 2 above)
Comment (samgnaniah) [reply en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md:372]: Should I remove all these three configurations?

File: en/mkdocs.yml (hunk 3 above)
Comment (samgnaniah) [reply en/mkdocs.yml]: Fixed.

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Changed this.

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md (hunk 2 above)
Comment (samgnaniah) [reply en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md:372]: Removed TM startup commands.

**Review by tharikaGitHub — APPROVED** [review]

**Pull request merged**